zerver/lib/push_notifications.py

```diff
@@ -774,8 +776,8 @@ def handle_push_notification(user_profile_id: int, missed_message: Dict[str, Any
     try:
         (message, user_message) = access_message(user_profile,
                                                  missed_message['message_id'],
-                                                 lock_message=False,
-                                                 lock_usermessage=False)
+                                                 lock_message=True,
+                                                 lock_usermessage=True)
```

Comment:
Is `lock_usermessage` sufficient?  I would expect so, because that's the only model we end up modifying, and the `Message` can't be deleted without deleted associated `UserMessage` rows.

Reply:
I verified via manual `manage.py shell` testing that my theory here was correct.